server/src/main/java/org/apache/iotdb/db/utils/datastructure/TVList.java

```diff
@@ -165,27 +167,27 @@ public void putVector(long time, Object[] value) {
     throw new UnsupportedOperationException(ERR_DATATYPE_NOT_CONSISTENT);
   }
 
-  public void putLongs(long[] time, long[] value, int start, int end) {
+  public void putLongs(long[] time, BitMap bitMap, long[] value, int start, int end) {
     throw new UnsupportedOperationException(ERR_DATATYPE_NOT_CONSISTENT);
   }
 
-  public void putInts(long[] time, int[] value, int start, int end) {
+  public void putInts(long[] time, BitMap bitMap, int[] value, int start, int end) {
     throw new UnsupportedOperationException(ERR_DATATYPE_NOT_CONSISTENT);
   }
 
-  public void putFloats(long[] time, float[] value, int start, int end) {
+  public void putFloats(long[] time, BitMap bitMap, float[] value, int start, int end) {
     throw new UnsupportedOperationException(ERR_DATATYPE_NOT_CONSISTENT);
   }
```

Comment:
How about change the order of the params?
Eg. `public void putFloats(long[] time, float[] value, BitMap bitMap, int start, int end) `

Reply:
Already Fixed. I also change params  order of relevant methods.